controllers/releaser_controller.go

```diff
@@ -52,6 +52,8 @@ import (
 type ReleaserReconciler struct {
 	client.Client
 	Scheme *runtime.Scheme
+
+	gitUser string
 }
 
 //+kubebuilder:rbac:groups=devops.kubesphere.io,resources=releasers,verbs=get;list;watch;create;update;patch;delete
```

Comment:
IMO, we should use rbac marker at here, please see https://book.kubebuilder.io/reference/markers/rbac.html.